lib/active_fedora/aggregation/base_extension.rb

```diff
@@ -29,11 +31,28 @@ def aggregates(name, options={})
         Builder.build(self, name, options)
       end
 
+      ##
+      # Create a association filter on the class
+      # @example
+      #   class Image < ActiveFedora::Base
+      #     aggregates :generic_files
+      #     filters_association :generic_files, as: :large_files, condition: :is_big?
```

Comment:
This is super nitpicky but I don't like encouraging people to name methods with `is_` ;)